answerking_app/tests/test_orders.py

```diff
@@ -76,7 +76,7 @@ def test_get_all_without_orders_returns_no_content(self):
         response = client.get("/api/orders")
 
         # Assert
-        self.assertEqual(response.status_code, 204)
+        self.assertEqual(response.status_code, 200)
```

Comment:
Here I changed the status to 200 because even though we are returning an empty list we are still returning something. So its technically not a 204.

Reply:
alright, then you'd need to add the lines to test the body.